backend/tests/test_exceptions_moodle.py

```diff
@@ -29,7 +29,7 @@ def test_non_manager_access_manager_endpoint(
         "instructor_firstname": "Franklin",
```

Comment:
Can we also add a test here to confirm that a non-authenticated user cannot access `GET /moodle/course/build`?